src/lib/components/UrlChooser/_styles.scss

```diff
@@ -0,0 +1,129 @@
+@import '../../../styles/settings/colors';
+@import '../../../styles/tools/mixins';
+
+// =============================================================================
+// URL CHOOSER
+//
+// Lets the user select a URL and trigger an audit.
+//
+// =============================================================================
+
+$border-radius: 6px;
+
+web-url-chooser {
+  $padding: 16px;
+  $cardsHeight: 219px;
+  $inputHeight: 56px;
+
+  display: block;
+  border-radius: $border-radius;
+
+  &[switching] {
```

Comment:
So I actually disagree with this somewhat. I'm happy to be overruled.

But this file has about ~15-20 rules, many of which are long (because of `::before`, `::-moz-focus-inner`, other specials). I've _flattened_ it a bit, but I think nesting still gains us a bit.

Additionally, the only way to have scoped variables in Sass is to nest. So we define `$INPUT_HEIGHT` at the top inside `web-url-chooser`, then use it here only.